.github/workflows/arduino.yml

```diff
@@ -17,12 +17,16 @@ jobs:
   build:
     # The type of runner that the job will run on
     runs-on: windows-latest
+    strategy:
+        matrix:
+            configuration: [Debug, Release]
```

Comment:
Do we really need the Debug configuration here? Can't we just use the Release one?

Reply:
I guess so.